src/settings/SettingsPage/SettingsPage.js

```diff
@@ -20,16 +20,52 @@ import {
   RoleDetails,
   SearchForm,
   useAuthorizationRoles,
+  useAuthorizationRolesMutation,
+  useRoleById,
+  useShowCallout,
   useUsers,
 } from '@folio/stripes-authorization-components';
 
 const SettingsPage = ({ path }) => {
+  const history = useHistory();
   const { id: roleId } = useParams();
+  const showCallout = useShowCallout();
 
   const [searchTerm, setSearchTerm] = useState('');
 
   const { roles, isLoading, onSubmitSearch } = useAuthorizationRoles();
-  const { users } = useUsers(roles.map(i => i.metadata.updatedByUserId));
+
+  const updatedByUserIds = useMemo(() => roles.map(i => i?.metadata.updatedByUserId), [roles]);
+  const { users } = useUsers(updatedByUserIds);
+
+  const {
+    duplicateAuthorizationRole,
+    isLoading: isDuplicating,
+  } = useAuthorizationRolesMutation();
+
+  const { roleDetails } = useRoleById(roleId);
+
+  const onDuplicate = () => {
+    const roleName = roleDetails?.name;
+    const messageIdPrefix = 'ui-consortia-settings.consortiumManager.members.authorizationsRoles.duplicate';
```

Comment:
Is role-duplication only available within ECS? If so, the menu item needs to be fenced off by an `<IfInterface>`. If not, then this translation needs to be moved from the `ui-consortia-settings` repo into this one.

Reply:
added translation message here

Reply:
It is available for both ECS and non-ECS